src/distributionPaybackMessage.js

```diff
@@ -0,0 +1,22 @@
+const { promises: fs } = require('fs')
+const Discord = require("discord.js")
+const client = new Discord.Client()
```

Comment:
starting a new client here wont work. Call this function from index.js and pass it the client through function params